shacl12-sparql/index.html

```diff
@@ -480,7 +583,16 @@ <h3>Document Conventions</h3>
 # This box represents an input shapes graph
 
 # Triples that can be omitted are marked as grey e.g.
-<span class="triple-can-be-skipped">&lt;s&gt; &lt;p&gt; &lt;o&gt; .</span>
+<span class="triple-can-be-skipped">&lt;s&gt; ex:p &lt;o&gt; .</span>
+					</div>
+					<div class="jsonld">
+						<pre class="text">// This box represents an input shapes graph</pre>
```

Comment:
JSON (not-LD) nit - I thought C-style comments were disallowed in JSON syntax?

Reply:
Oh, nevermind, I got confused by the `class` attribute on the `div` vs. the `pre` below.  All's well.